src/kit/Tooltip.tsx

```diff
@@ -31,11 +31,18 @@ const Tooltip: React.FC<Props> = ({
   mouseEnterDelay,
   open,
   placement = 'top',
+  trigger,
   ...props
 }: Props) => {
+  const triggers = useMemo<TooltipProps['trigger']>(
+    () => [trigger || 'hover', 'focus'] as string[],
```

Comment:
consider refining this so it only adds `focus` to the trigger list if the only trigger is `hover`. `click` and `contextmenu` are valid triggers too, but are already accessible.

in addition, you need to make sure the child component is focusable.